src/main/java/org/zalando/nakadi/domain/EventPublishingStep.java

```diff
@@ -4,6 +4,7 @@
     NONE,
     VALIDATING,
     ENRICHING,
+    VALIDATING_SIZE,
```

Comment:
Unfortunately you can't change this. Because this enum is defined in our API and if you add values here that will mean that we add backwards incompatible changes.
https://github.com/zalando/nakadi/blob/master/api/nakadi-event-bus-api.yaml#L1817

Why do you want a separate step for that? I thought I can be part of VALIDATING step.